### DISCUSSION TITLE
reagarding submission

### DISCUSSION THREAD

**Discussion by Am121000** [pykt-team/pykt-toolkit#66] [Q&A] [answered]
I am unable to submit the prediction.csv file
![Screenshot_20221111_190958](https://user-images.githubusercontent.com/83466056/201352067-c7da2333-881a-4e9a-8ad5-437d62cb92dc.png)

**Comment by meta-tabchen** [comment, marked as answer]
Hello, we check your submission file, the problem is the zip format. You can check your file to make sure it isn’t a zip folder that includes the prediction.csv.
